### PR TITLE
Update api.rst

### PR DESCRIPTION
**Pull request by ksIsCute**
idk rst very well but i need to know where i can edit and make descriptions for the payloads and stuff

## Summary
editing rst so i can autoclass this later and add onto the attributetable

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, …)

### CODE REVIEW

**Comment by EnokiUN** [comment]
That's not how you do that.

File: docs/source/api.rst [hunk 1]
```diff
@@ -199,3 +199,6 @@ Enums
 
 .. autoclass:: voltage.RelationshipType
 
+Event Listeners
+===============
+.. attributetable:: voltage.Client.listen
```
Comment (ooliver1) [inline docs/source/api.rst:204]: is listen a class? and its just the method so it wont automatically list every event

**Review by ooliver1 — CHANGES_REQUESTED** [review]

File: docs/source/api.rst (hunk 1 above)
Comment (EnokiUN) [reply docs/source/api.rst:204]: it's a method that's already documented because client is automatically documented

Comment (EnokiUN) [reply docs/source/api.rst:204]: what you're *supposed* to do is add a new section for listeners right under the client section and provide a list of the listener's names, their corresponding api events, maybe a simple explanation and the arguments they return

**Pull request closed without merging**